templates/bundles/PrestaShopBundle/Admin/Configure/AdvancedParameters/Webservice/index.html.twig

```diff
@@ -52,6 +52,45 @@
     {% endif %}
   {% endblock %}
 
+  {% block webservice_api_status %}
+    {% set devdocUrl = 'https://devdocs.prestashop.com/1.7/development/webservice/' %}
+
+    {% if webserviceStatus['enabled'] == true %}
+      <div class="card">
+        <h3 class="card-header">
+          <i class="material-icons">info_outline</i> {{ 'Webservice status'|trans }}
+        </h3>
+        <div class="card-block">
+          <div class="card-text">
+            <p>
+              {{ 'Webservice is enabled. Main entry point is' |trans }}
+              <a href="{{ webserviceStatus['endpoint'] }}">
+                {{ webserviceStatus['endpoint'] }}
+              </a>
+            </p>
+            {% if webserviceStatus['isFunctional'] == true %}
+              <p>
+                {{ "Read the [1]developer documentation[/1] to learn more about it."|trans({
```

Comment:
What does "it" refer to? If it refers to "webservice" maybe it would be better to say "To learn more about webservice, read the [1]developer documentation[/1]."